src/nr_openai_observability/langchain_callback.py

```diff
@@ -86,14 +96,14 @@ def on_llm_end(self, response: LLMResult, **kwargs: Any) -> Any:
                     "total_tokens": token_usage.get("total_tokens", None),
                 }
             )
-        self._finish_segment(kwargs["run_id"])
+        self._finish_segment(kwargs["run_id"], tags, CompletionEventName)
```

Comment:
It looks like you're re-purposing the `LlmCompletion` event here. I think we technically use that in the baseline OpenAI instrumentation as well. Did you want to namespace this to be more langchainy? Slash are you worried about those two types getting jumbled?

Reply:
I am always open to a new name. My thought is that since the `LlmCompletion` is the overall completion, having one from LangChain could/would act as a wrapper around any number of model/LLM calls. They would all still have the same agent transactionId/traceId.